### PR TITLE
[AIRFLOW-2489] Update dependencies

### PR DESCRIPTION
**Pull request by Fokko**
Make sure you have checked _all_ steps below.

The Apache Airflow CI cache has been bugged because of the downgrade of the Flask dependency: https://github.com/dpgaspar/Flask-AppBuilder/commit/e5d292bba8e5623522940349301f845d791f5bf8

### JIRA
- [x] My PR addresses the following [Airflow JIRA](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-2489\] My Airflow PR"
    - https://issues.apache.org/jira/browse/AIRFLOW-2489
    - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-2489\], code changes always need a JIRA issue.

### Description
- [x] Here are some details about my PR, including screenshots of any UI changes:

### Tests
- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits
- [x] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

### Documentation
- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
    - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.

### Code Quality
- [ ] Passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -248,7 +248,7 @@ def do_setup():
             'configparser>=3.5.0, <3.6.0',
             'croniter>=0.3.17, <0.4',
             'dill>=0.2.2, <0.3',
-            'flask>=0.12, <0.12.2',
+            'Flask<0.12.2,>=0.10.0',
```
Comment (kaxil) [inline setup.py]: Is there any specific reason for `>=0.10.0`?

Both of them are going to install `0.12.1` version right? Let me know if I am missing something?

**Comment by kaxil** [comment]
LGTM now. I just saw `Flask-AppBuilder` released `v1.11.1`.  👍 ☕️

**Comment by Fokko** [comment]
@kaxil PTAL

This should fix the version issues with FlaskAppBuilder hopefully: https://github.com/dpgaspar/Flask-AppBuilder/commit/e5d292bba8e5623522940349301f845d791f5bf8#commitcomment-29045752

Still a lot of failing tests..

**Comment by kaxil** [comment]
Yes, there is one specific tests `dag_run_catchup` which fails randomly.

**Pull request closed without merging**